mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -2586,6 +2617,16 @@ int main(int argc, char **argv) {
         genParams.convConfig = llvm::None;
         genParams.arch = arch;
         (void)createGpuGemmKernel(module, genParams);
+        if (emitTuningSpace) {
```

Comment:
So, can we move this out of this if, so that this can work for the convolutions as well.
The original ticket was to add a script for the gemm but no harm to enable it for conv